recipes/plumpy/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "plumpy" %}
+{% set version = "0.7.9"%}
```

Comment:
Is there a reason to go with `0.7.9` rather than `0.9.1`, which is the newest release? Actually now that I look at it, there doesn't appear to be any files on pypi.io for `0.9.1`.